src/scicookie/hooks/post_gen_project.py

```diff
@@ -291,6 +306,14 @@ def add_binding_source_files():
         else:
             os.makedir(src_system_dir)
             shutil.move(build_system_dir / "lib.rs", src_system_dir)
+    elif BUILD_SYSTEM == "pybind11" :
+        build_system_dir = PROJECT_DIRECTORY / "build-system"
+        src_system_dir = PROJECT_DIRECTORY/ "src"
```

Comment:
I think `PROJECT_DIRECTORY` automatically checks for flat or `src` layout?

Reply:
I think it for `main.cpp` which should be in `src` directory like this. 
your_project/
|-- setup.py
|-- pyproject.toml
|-- src/
|   |-- main.cpp
|   |-- other_cpp_files.cpp
|   |-- your_module_name_here.cpp
|   |-- your_module_name_here.h
|-- your_package/
|   |-- __init__.py
|   |-- your_module_name_here.py
|-- tests/
|   |-- test_your_module_name_here.py
|-- README.md
|-- LICENSE

Reply:
this is an interesting discussion ... maybe we can return to this discussion to check how we should proceed not just for this build-system but maybe for all build-systems.
also maybe in our repo, maybe we could have this files in folders called by the programming language (`cpp`, `rust`, etc)